dist/documentation/themes/dark.js

```diff
@@ -163,6 +163,59 @@ primary : {textBase:{
             hex: "#1A5740",
             description: "Used for the pressed border colour for primary components such as buttons"
           }},
+success : {textBase:{
+            name: "success.textBase",
+            hex: "#42AD86",
+            description: "Used as the text or fill colour for interactive and static primary components such as links, text, and icons"
```

Comment:
Let's update all the descriptions to say `success` instead of `primary`!

Reply:
Updated all of these in `dark`! (Also a few I missed in `light`)